src/workspace-data/Data.js

```diff
@@ -1170,10 +1170,13 @@ export const WorkspaceData = _.flow(
                   uploadingFile &&
                     h(EntityUploader, {
                       onDismiss: () => setUploadingFile(false),
-                      onSuccess: () => {
+                      onSuccess: (recordType) => {
                         setUploadingFile(false);
                         forceRefresh();
                         loadMetadata();
+                        notify('success', `Data imported successfully to table ${recordType}.`, {
```

Comment:
This always shows the message immediately - even for an async upload that takes a while.  For async uploads, it shouldn't show a success message until it's actually complete.  Async uploads already show a success message upon completion, so with this change we get two success messages